modules/android/pages/_partials/pn-issues-list.adoc

```diff
@@ -1,6 +1,47 @@
 // BEGIN: tagged-inclusion -- issues list for Android
 == Issues in Release 2.8
 
+
+// BEGIN 2-8-6
+
+=== 2.8.6
+
+// tag::issues-2-8-6[]
+// ==== Deprecated
+// tag::deprecated-2-8-6[]
+// end::deprecated-2-8-6[]
+
+==== Enhancements
+// tag::enhancements-2-8-6[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=enhancements-2-8-6]
+
+* https://issues.couchbase.com/browse/CBL-1979[CBL-1979] -- Android v30 compatibility
```

Comment:
This was inadvertently tagged for this release. Only CBL-2001 should be here

Reply:
Removed